### ISSUE TITLE
"the oldstyle protocol is no longer supported" from Image Explorer.

### ISSUE THREAD

**Issue by FluorescentGreen5** [rescuezilla/rescuezilla#167]
I made a backup of my OS drive (with one of the partitions excluded) recently and decided to test out the image explorer. However, I got the following error when I tried to mount it:
```
Failed to run command: nbd-client-block-size 512 localhost /dev/nbd0

Warning: the oldstyle protocol is no longer supported.
This method now uses the new style protocol with a default export
Error: Socket failed: Connection refused
```

**Comment by shasheene** [comment]
You can ignore the message "Warning: the oldstyle protocol is no longer supported. This method now uses the new style protocol with a default export". It's a warning message `nbdkit` always produces even on success. I'll try and suppress it because it's not actually indicative on any problem.

The issue here is the "Error: Socket failed: Connection refused" error. In the past I have found:

1. Sometimes clicking mount again resolves this issue.
2. Configuring network/internet seems to be required for successful connection to the server hosted on "localhost"

By the way, as you may have read in [task #153](https://github.com/rescuezilla/rescuezilla/issues/153), the Image Explorer (beta) is currently unusable slow for any image that's not an uncompressed image created by Clonezilla's Expert Mode. For compressed images it might take 10+ minutes to mount your image, but then even after mounting it will be way too slow to navigate. I have some ideas on how to improve this that I'll implement for Rescuezilla v2.2.

**Comment by shasheene** [comment]
Oh, this issue has been fixed in [Rescuezilla v2.2](https://github.com/rescuezilla/rescuezilla/releases/latest) by polling the NBD device until it's ready.
